layers/meta-balena-jetson/conf/machine/skx2.conf

```diff
@@ -0,0 +1,8 @@
+#@TYPE: Machine
+##@NAME: skx2
+##@DESCRIPTION: Machine configuration for the SKX2
+
+MACHINEOVERRIDES = "jetson-tx2:${MACHINE}"
+include conf/machine/jetson-tx2.conf
+
+PREFERRED_VERSION_linux-tegra = "4.4.38.skx2%"
```

Comment:
Kernel in this repo is 4.9, can be dropped I guess.

Reply:
Indeed, I will remove that. Thanks